### PR TITLE
ARCH-873 - Fixing some problems with the templates.

### PR DESCRIPTION
**Pull request by danielle-casella-adams**

### CODE REVIEW

File: workflow-templates/im-deploy-tf-manually-apply-to-prod-subscription.yml [hunk 1]
```diff
@@ -161,6 +153,9 @@ jobs:
 
           echo "::set-output name=tf_plan_name::$terraformPlanName"
 
+      - name: Azure logout
+        run: az logout
```
Comment (JosephDSchwartz) [inline workflow-templates/im-deploy-tf-manually-apply-to-prod-subscription.yml:157]: The azure/login action recommends doing `az cache purge` and `az account clear` in addition to `az logout` (https://github.com/marketplace/actions/azure-login#az-logout-and-security-hardening). Is that something we want to do or nah? I don't know enough about azure and our runners to know if it's a good idea or not, but I thought I'd ask.

**Review by JosephDSchwartz — DISMISSED** [review]
I have the one question, but it may be nothing so I'll go ahead and approve!

File: workflow-templates/im-deploy-tf-manually-apply-to-prod-subscription.yml (hunk 1 above)
Comment (danielle-casella-adams) [reply workflow-templates/im-deploy-tf-manually-apply-to-prod-subscription.yml]: I didn't know that was a thing.  I will add it.

**Review by JosephDSchwartz — APPROVED** [review]
Your commit message legitimately made me laugh out loud 🤣

**Review by hpractv — APPROVED** [review]
👍🏽

**Pull request merged**